content/docs/lifting-state-up.md

```diff
@@ -299,32 +323,48 @@ class Calculator extends React.Component {
+Het leert hoe hun gebruikersinterface eruit moet zien.
+* React roept de `render` methode van de `BoilingVerdict` component aan, waarbij de temperatuur in Celsius als prop wordt doorgegeven.
+* React DOM werkt de DOM bij met het boiling verdict en zodat de gewenste input waarden getoond worden.
+De input die we zojuist hebben bewerkt, ontvangt de huidige waarde en de andere input wordt bijgewerkt naar de temperatuur na de conversie.
 
-Every update goes through the same steps so the inputs stay in sync.
+Elke update gaat door deze zelfde stappen zodat de inputs gesynchroniseerd blijven.
 
-## Lessons Learned {#lessons-learned}
+## Les Geleerd {#lessons-learned}
 
-There should be a single "source of truth" for any data that changes in a React application. Usually, the state is first added to the component that needs it for rendering. Then, if other components also need it, you can lift it up to their closest common ancestor. Instead of trying to sync the state between different components, you should rely on the [top-down data flow](/docs/state-and-lifecycle.html#the-data-flows-down).
+Er moet een enkele "bron van waarheid" zijn voor alle gegevens die veranderen in een React applicatie.
+Gewoonlijk wordt de staat eerst toegevoegd aan de component die deze nodig heeft voor zijn weergave.
+Als dan andere componenten ze ook nodig hebben, kan je ze omhoog tillen naar hun meest nabije gemeenschappelijke voorvader.
+In plaats van de staat tussen verschillende componenten proberen te synchroniseren, zou je moeten vertrouwen op de [top-down data flow](/docs/state-and-lifecycle.html#the-data-flows-down).
 
-Lifting state involves writing more "boilerplate" code than two-way binding approaches, but as a benefit, it takes less work to find and isolate bugs. Since any state "lives" in some component and that component alone can change it, the surface area for bugs is greatly reduced. Additionally, you can implement any custom logic to reject or transform user input.
+Het tillen van staat houdt het schrijven van meer "boilerplate" code in dan bidirectionele bindingsbenaderingen, maar als voordeel, kost het minder moeite om bugs to vinden en te isoleren.
+Aangezien elke staat in een bepaalde component "leeft" en alleen die component ze kan veranderen, wordt de kans op fouten aanzienlijk verminderd.
+Bovendien kun je aangepaste logica implementeren om gebruikersinvoer te weigeren of te transformeren.
 
-If something can be derived from either props or state, it probably shouldn't be in the state. For example, instead of storing both `celsiusValue` and `fahrenheitValue`, we store just the last edited `temperature` and its `scale`. The value of the other input can always be calculated from them in the `render()` method. This lets us clear or apply rounding to the other field without losing any precision in the user input.
+Als iets kan afgeleid worden van props of state, zou het waarschijnlijk niet thuis horen in de state.
```

Comment:
zou het  waarschijnlijk niet thuis horen in de state => hoort het waarschijnlijk niet thuis in de state